test/integration/singlecluster/tas/tas_test.go

```diff
@@ -681,6 +681,26 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 					))
 				})
 			})
+			ginkgo.It("should skip workloads with missing ClusterQueue after restart", func() {
+				badWl := testing.MakeWorkload("bad-wl", ns.Name).
+					Queue("non-existent-queue").
+					PodSets(*testing.MakePodSet("main", 1).
+						Request(corev1.ResourceCPU, "1").
+						Obj()).
+					Obj()
+				util.MustCreate(ctx, k8sClient, badWl)
+				ginkgo.By("Restarting Kueue manager", func() {
+					fwk.StopManager(ctx)
+					fwk.StartManager(ctx, cfg, managerSetup)
+				})
+
+				ginkgo.By("Verifying workload stays pending", func() {
+					gomega.Consistently(func() bool {
+						return k8sClient.Get(ctx, client.ObjectKeyFromObject(badWl), badWl) == nil &&
```

Comment:
Please avoid using this solution. Instead, propagate gomega.Gomega as shown below:

```go
gomega.Consistently(func(g gomega.Gomega) {
...
, util.ConsistentDuration, util.Interval).Should(gomega.Succeed())
```